spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu'
+
+describe Menu do
+
+  let(:biriyani){double :biriyani, title: "Biriyani", price: 5.00}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.